packages/stryker-jasmine-runner/src/JasmineTestRunner.ts

```diff
@@ -0,0 +1,70 @@
+import { EOL } from 'os';
+import { TestRunner, RunResult, TestResult, RunStatus } from 'stryker-api/test_runner';
+import { StrykerOptions } from 'stryker-api/core';
+import { Jasmine, toStrykerTestResult, evalGlobal } from './helpers';
+
+export default class JasmineTestRunner implements TestRunner {
+
+  private jasmineConfigFile: string | undefined;
+  private fileNames: ReadonlyArray<string>;
+  private Date: { new(): Date } = Date; // take Date prototype now we still can (user might choose to mock it away)
+
+  constructor({ fileNames, strykerOptions: { jasmineConfigFile } }: { fileNames: ReadonlyArray<string>, strykerOptions: StrykerOptions }) {
+    this.jasmineConfigFile = jasmineConfigFile;
+    this.fileNames = fileNames;
+  }
+
+  run(options: { testHooks?: string }): Promise<RunResult> {
```

Comment:
Why do you specify the type here like this instead of using `RunOptions`?

Reply:
See my previous remark.

Reply:
I would like to keep this code. We ignore the timeout setting.